vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc

```diff
@@ -9,7 +9,9 @@
 #include "bat/confirmations/internal/confirmations_impl.h"
 #include "bat/confirmations/internal/unblinded_tokens.h"
 #include "bat/confirmations/internal/redeem_payment_tokens_request.h"
+
```

Comment:
nit: can you remove it?

Reply:
This space is to separate non bat-native-confirmations includes